src/guess/guess.html

```diff
@@ -10,16 +10,16 @@
    <label for="guess">Your Guess:</label>
    <input type="text" id="guess" value="" placeholder="0 - 100" />
    <input type="button" onclick="yourGuess()" value="submit" /><br />
-   <input type="button" onclick="" value="Show My Guesses" />
-   <input type="button" onclick="" value="New Game" /><br />
+   <input type="button" id="showguesses" onclick="showGuesses()" value="Show My Guesses" />
```

Comment:
This is the coolest version of the guess game I've seen so far! I think that initially, it should be `value="Hide My Guesses"`